docs/installation/nuxt.md

```diff
@@ -35,7 +35,7 @@ npm install @tiptap/vue-2 @tiptap/starter-kit
 If you followed step 1 and 2, you can now start your project with `npm run serve`, and open [http://localhost:8080/](http://localhost:8080/) in your favorite browser. This might be different, if you’re working with an existing project.
 
 ## 3. Create a new component
-To actually start using Tiptap, you’ll need to add a new component to your app. Let’s call it `Tiptap` and put the following example code in `src/components/Tiptap.vue`.
+To actually start using Tiptap, you’ll need to add a new component to your app. Let’s call it `TipTap` and put the following example code in `~/components/TipTap.vue`.
```

Comment:
`~` is an alias for the user home directory, so if someone is created the file from the CLI it would end up in the wrong location.

Can you just write `components/TiptapEditor` here - without the `~`?

Reply:
In a Nuxt project configuration `~` refers to the root folder of the project. In contrast to a vue project created with vue-cli project, the official Nuxt cli project creator (create-nuxt-app) uses this notation out of the box.

Reply:
Yeah, I know. But if some says "create a file", which you normally do on your file system, editor, the bash ... in that context it stands for a different folder, the user home.

If we would say "import that component" (which then would happen in the Nuxt context) that would be a different thing, and would make sense.

Reply:
I see, didn't realize it was a wording problem. Then `components/TiptapEditor.vue` should do just fine.